### DISCUSSION TITLE
Option to show all nonzero precipitation chance values in hourly forecast

### DISCUSSION THREAD

**Discussion by Slusho64** [breezy-weather/breezy-weather#1255] [Ideas]
### Describe your idea

It seems that right now in the hourly forecast, precipitation chance is only shown when above a certain threshold (not just a nonzero value but something like 5%). If I am wrong about this let me know. Other apps I've used show anything 1% or higher. Consequently, yesterday I left the house without an umbrella after checking the weather and subsequently got rained on. :( I would like the option to show any nonzero precipitation chance values in the hourly forecast.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.7 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Indeed, % is only shown when ≥ 5. It's a legacy behavior, I have no idea why it was done that way, I willl investigate. I'm not sure users will change their behavior when they see a 1-4 % chance of rain though 😅

Will track investigation in #1256

**Comment by papjul** [comment]
The rule will be removed in v5.2.8.
